src/corg/developer-environment-setup/getting-the-nuxeo-source-code.md

```diff
@@ -225,75 +225,100 @@ Starting from version 5.9.2, Nuxeo requires Maven <span style="color: rgb(34,34,
 
 Contributing to Nuxeo **does not** require cloning Nuxeo source code: see [Contributing to Nuxeo]({{page space='nxdoc' page='contributing-to-nuxeo'}})
 
 {{/callout}}
 
-### Import Nuxeo Source Code in Eclipse
-
-Of course for the dependencies to be resolved in Eclipse they will have to be loaded first in your Maven repository, which you can do by performing at least one build first:
+After having cloned the Nuxeo repository, you need to run a full build in Maven to download all the dependencies.
 
 {{#> panel type='code' heading='Full build Nuxeo without running tests'}}
 
 ```
-export MAVEN_OPTS="-Xmx4096m -Xms1024m -XX:MaxPermSize=512m"
+export MAVEN_OPTS="-Xmx4096m -Xms1024m"
 mvn -DskipTests install -Paddons
-(cd nuxeo-distribution ; mvn -DskipTests install)
+cd nuxeo-distribution
+mvn -DskipTests install
 ```
 
 {{/panel}}
 
-If you want to open/edit the projects in Eclipse do the following at the very root of the Git repository:
+Please go back to root of the Nuxeo folder once the build is finished.
+
+### Import Nuxeo Source Code in Eclipse
+
+Before importing Nuxeo source code in Eclipse, you need to generate Eclipse project files in terminal. Compared to doing
+it in Eclipse, using the command-line makes the process much faster. Please launch the following command at the root of the
+Git repository. All the properties, starting with `-D`, are optional, you can remove them according to your use-case.
+
+{{#> callout type='note' }}
+
+Don't forget to replace the expression `/path/to/eclipse/workspace` to your real workspace.
+
+{{/callout}}
 
 {{#> panel type='code' heading='Generate Eclipse\'s project files'}}
 
 ```
-# For details, see Maven Eclipse Plugin documentation: https://maven.apache.org/plugins/maven-eclipse-plugin/eclipse-mojo.html
-mvn eclipse:clean eclipse:eclipse -Paddons,distrib [-DdownloadSources=true] [-DdownloadJavadocs=true] [-Declipse.useProjectReferences=true] [-Declipse.workspace=/path/to/eclipse/workspace] -fae -nsu -T 1C
-# Linux and OS X users should then run the following script to make Eclipse use different build directories than Maven:
-./fixeclipse
-# A cross-platform script is also available for Windows users:
-# python scripts/fixeclipse.py
+mvn eclipse:clean eclipse:eclipse \
+    -Paddons,distrib \
+    -DdownloadSources=true \
+    -DdownloadJavadocs=true \
+    -Declipse.useProjectReferences=true \
+    -Declipse.workspace=/path/to/eclipse/workspace \
+    -fae -nsu -T 1C
 ```
 
 {{/panel}}
 
-Then in Eclipse:
+{{#> callout type='note' }}
 
-{{#> panel type='code' heading='Import Nuxeo modules into Eclipse'}}
+For details, see [Maven Eclipse Plugin documentation](https://maven.apache.org/plugins/maven-eclipse-plugin/eclipse-mojo.html).
+
+{{/callout}}
+
+You should then run the following script to make Eclipse use different build directories than Maven:
+
+{{#> panel type='code' heading='Linux and OS X'}}
 
 ```
-File -> Import -> General -> Existing Projects into Workspace
+./fixeclipse
```

Comment:
I think this would be useful in <https://doc.nuxeo.com/corg/maven-usage/>. What do you think?